routes/api/db/contactsSchema.js

```diff
@@ -0,0 +1,29 @@
+// const mongoose = require("mongoose");
```

Comment:
Nie trzymaj zakomentowanego kodu. Jeśli będziesz chciał do czegoś wrócić to możesz się gitem cofnąć do commita, lub nawet usunąć zmiany które wprowadził.